system/lib/libc/musl/src/thread/pthread_attr_get.c

```diff
@@ -44,7 +44,7 @@ int pthread_attr_getstack(const pthread_attr_t *restrict a, void **restrict addr
 //		return EINVAL;
 
 	*size = a->_a_stacksize + DEFAULT_STACK_SIZE;
-	*addr = (void *)(a->_a_stackaddr - *size);
+	*addr = (void *)(a->_a_stackaddr);
 	return 0;
```

Comment:
I don't understand this change, but then I don't understand the code from before either I think :)

This change is for both fastcomp and the wasm backend, it seems, since it's not ifdef'd? So was this always wrong?

@juj you know this code the best, what do you think?

Reply:
I'm confused. The base musl implementation does the same thing:
https://github.com/kraj/musl/blob/cbf41860664b2f408653cc2169b454a293096ccf/src/thread/pthread_attr_get.c#L38-L45

and the pthread_attr_getstack doco says:

> [The base (lowest addressable byte) of the storage shall be addr](https://pubs.opengroup.org/onlinepubs/009695399/functions/pthread_attr_getstack.html)

So maybe I'm wrong, but the the current behaviour looks right?
What bug is this trying to fix?